lib/configurators/aws/aws_test.go

```diff
@@ -1317,6 +1318,13 @@ type SSMMock struct {
 func (m *SSMMock) CreateDocumentWithContext(ctx aws.Context, input *ssm.CreateDocumentInput, opts ...request.Option) (*ssm.CreateDocumentOutput, error) {
 
 	m.t.Helper()
+
+	// UUID's are unpredictable, so we remove them from the content
+	uuidRegex := regexp.MustCompile(`[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}`)
```

Comment:
Cache the regex? Go 1.21 can use [sync.OnceValue](https://pkg.go.dev/sync#OnceValue), older versions can do something like this:

(top-level declaration)

```go
var uuidRegex = struct {
	value *regexp.Regexp
	once  sync.Once
}{}
```

and then:

```go
	uuidRegex.once.Do(func() {
		uuidRegex.value = regexp.MustCompile(`[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}`)
	})
```

Reply:
Is it worth the (admittedly minor) complexity for testing?

Reply:
sync.OnceValue is pretty cheap, which is how I started. Your call.